### DISCUSSION TITLE
🔧 Looking for maintainers!

### DISCUSSION THREAD

**Discussion by sanny-io** [sanny-io/react-appwrite#1] [Announcements]
It takes a lot of effort to develop and test this project, and there are even more Appwrite features on the horizon. Anyone who would like to help, feel free to post here 😊

**Comment by antonrejoe** [comment]
A guy here , but I'm quite new to appwrite if I get proper guidance I think I could make it

**Reply by sanny-io** [reply]
The Appwrite documentation and React Query docs should be everything you need. The code in this repository also follows quite a similar pattern from one hook to the other.

https://tanstack.com/query/v4/docs/react/guides/queries
https://appwrite.io/docs

I've added you to the repo 👍

**Comment by dishwasher-detergent** [comment]
Hey! I'd love to help with this project.

**Reply by sanny-io** [reply]
Alright! I've added you to the repo 🤙

**Comment by sanny-io** [comment]
Also, apologies I haven't made much commits this past week or so. Been busy with a couple other things.

I will do a bunch of stuff this weekend.

**Comment by probablyArth** [comment]
I love appwrite and react, will try my best to help in my free time!

**Reply by sanny-io** [reply]
Invited you 🤙

**Comment by sanny-io** [comment]
I've invited everyone to a new `react-appwrite` organization. I will likely transfer the repository there in the near future. Everything will remain intact, so don't worry about losing anything.

@probablyArth 
@dishwasher-detergent 
@antonrejoe

**Comment by vineetp6** [comment]
Would love to help , please add.

**Reply by sanny-io** [reply]
Sure. Invited you.

**Comment by stephane2018** [comment]
I would like contribute on this projet , please add me
